### DISCUSSION TITLE
Wazuh Readonly user with group

### DISCUSSION THREAD

**Discussion by PixiesFR** [wazuh/wazuh#26016] [Q&A]
Hello all,

We use actually wazuh 4.9 latest release.
After upgrade to 4.9, we get an issue on read only user with group.

When we login with the user , we see the dashboard in readonly access with the limitation on the group applied : no issue.
when we logout and login again, then the user have full admin access on the dashboard, even if we clean the cookies and data site.

any help will be appreciate :)

Thank you
Fabrice.

**Comment by patheticode** [comment]
same problem, does anyone have a solution other than downgrading?
